views/scss/qti/_qti-base.scss

```diff
@@ -117,6 +117,10 @@ $mainContainer : '.qti-item';
                 background: $errorBgColor;
             }
         }
+
+        &.highlighter-cursor {
+            cursor: url("../img/qtiIconsPng/marker.png") 0 32, auto;
```

Comment:
do you think this is the right way to add icons?
I know only this way: https://hub.taotesting.com/styleguide/icons (don't know if anything changed)

Reply:
it is not an icon, but static asset